src/types.ts

```diff
@@ -75,12 +75,17 @@ export interface DashboardWidgetContext {
 export interface MenuItemContext {
     key: string;
 }
+// Generic type of optional arguements passed to different feature components like modal, sidepanel, etc
+export interface CustomFeatureContext {
+    [key: string]: any;
+}
 
 // A combined object specifing data about the context of a feature within the app
 export interface FeatureContext {
     dashboard?: DashboardContext;
     widget?: DashboardWidgetContext;
     menuItem?: MenuItemContext;
+    sidePanel?: CustomFeatureContext;
```

Comment:
Do you need to update this to match the change in web-ui?

Reply:
ah I made the change in `/dist` instead :). Thanks for catching this. Fixed.